architecture/index.md

```diff
@@ -1,10 +1,60 @@
 ---
 layout: page
 title: DE Architecture 
-root: ../
 ---
 
-_*Coming Soon*_
+# Infrastructure
 
-* [Databases](databases)
+The following sections describe the components of the infrastructure on which the DE operates.
+
+## Data Store
+The DE is provides access and management of data via the CyVerse 
+[Data Store](http://www.cyverse.org/ci/data-store), which is built on top of 
+[iRODS](http://irods.org/).
+
+## Compute Platform(s)
+The DE integrates the Data Store with [HTCondor](https://research.cs.wisc.edu/htcondor/) and the
+[Agave Platform](http://agaveapi.co/) to provide a large set of tools for performing resource
+intense analyses.
+
+## PostgreSQL
+Nearly all application use a database. The DE uses [PostgreSQL](http://www.postgresql.org/). The 
+schemas can be found 
+[here]({{site.github.repository_url}}/tree/master/databases).
+
+## RabbitMQ
+[RabbitMQ](https://www.rabbitmq.com/) is used throughout our services, but primarily to integrate 
+our services with iRODS.
+
+## Elasticsearch
+The DE uses [Elasticsearch](https://www.elastic.co/products/elasticsearch) to provide search and 
+other capabilities.
+
+## Docker
+[Docker](https://www.docker.com/) is used throughout the DE architecture. Most importantly, all of 
+the tools used on the compute platform are packaged as Docker images. 
```

Comment:
Not sure if we want to mention that the backwards-compat image still requires scripts mounted from a shared file system or not.